### ISSUE TITLE
a11y: Package download table

### ISSUE THREAD

**Issue by woodruffw** [pypa/warehouse#6081]
Minor accessibility improvements can be made to `/project/{project_name}/#files`:

* The `<table>` itself should have a `<caption>`, as the first child node. This is partially mitigated by the `<h2>` above
* The `<table>` should include `scope=col` or another appropriate scope for each `<th>` node

**Comment by woodruffw** [comment]
The "copy to clipboard" link (see below) is smaller than in other tables, and might be difficult for users with limited mobility to click.

For improved accessibility and consistency with other instances of clipboard actions, the link should be turned into a button and made larger and/or placed into its own column on the download table.

<img width="873" alt="Screen Shot 2019-06-24 at 10 46 54 AM" src="https://user-images.githubusercontent.com/3059210/60028495-66b12d80-966d-11e9-9fc5-6fe17e172cd8.png">

**Comment by woodruffw** [comment]
The font size on the table can also be increased from `14px`, potentially to `17px` (for consistency with other tables in the project view).
